en/micro-integrator/docs/references/connectors/smpp-connector/smpp-connector-config.md

```diff
@@ -0,0 +1,644 @@
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>addressNpi</td>
+            <td>Numbering Plan Indicator for ESME address.</td>
+            <td>Optional</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <SMPP.init>
+        <host>{$ctx:host}</host>
+        <port>{$ctx:port}</port>
+        <systemId>{$ctx:systemId}</systemId>
+        <password>{$ctx:password}</password>
+        <enquireLinkTimer>{$ctx:enquireLinkTimer}</enquireLinkTimer>
+        <transactionTimer>{$ctx:transactionTimer}</transactionTimer>
+        <systemType>{$ctx:systemType}</systemType>
+        <addressTon>{$ctx:addressTon}</addressTon>
+        <addressNpi>{$ctx:addressNpi}</addressNpi>
+    </SMPP.init>
+    ```
+    
+    **Sample request**
+    
+    Following is a sample REST/JSON request that can be handled by the init operation.
+    ```json
+    {
+      "host": "127.0.0.1",
+      "port": 2775,
+      "systemId": "DAMIEN",
+      "password": "neimad",
+      "systemType": "UNKNOWN",
+      "addressTon": "UNKNOWN",
+      "addressNpi": "UNKNOWN",
+      "enquireLinkTimer": "50000",
+    }
+    ```
+## Send SMS Message
+
+??? note "sendSMS"
+    Use to send SMS Message to the SMSC (Short Message Service Center),
+    <table>
+    <tr>
+       <th>Parameter Name</th>
+       <th>Description</th>
+       <th>Required</th>
+    </tr>
+    <tr>
+       <td>serviceType</td>
+       <td>
+          Indicates SMS application service. The following generic service_types are defined:
+          <table>
+             <tr>
+                <td>"" (NULL)	</td>
+                <td>Default</td>
+             </tr>
+             <tr>
+                <td>"CMT"</td>
+                <td>Cellular Messaging</td>
+             </tr>
+             <tr>
+                <td>"CPT"	</td>
+                <td>Cellular Paging</td>
+             </tr>
+             <tr>
+                <td>"VMN"</td>
+                <td>Voice Mail Notification</td>
+             </tr>
+             <tr>
+                <td>"VMA"</td>
+                <td>Voice Mail Alerting</td>
+             </tr>
+             <tr>
+                <td>"WAP"</td>
+                <td>Wireless Application Protocol</td>
+             </tr>
+             <tr>
+                <td>"USSD"</td>
+                <td>Unstructured Supplementary Services Data</td>
+             </tr>
+          </table>
+       </td>
+       <td>Optional</td>
+    </tr>
+    <tr>
+       <td>sourceAddressTon</td>
+       <td>Type of number for source address.</td>
+       <td>Optional</td>
+    </tr>
+    <tr>
+       <td>sourceAddressNpi</td>
+       <td>Numbering plan indicator for source address.</td>
+       <td>Optional</td>
+    </tr>
+    <tr>
+       <td>sourceAddress</td>
+       <td>Source address of the SMS message.</td>
+       <td>Yes</td>
+    </tr>
+    <tr>
+       <td>destinationAddressTon</td>
+       <td> type of number for destination.</td>
```

Comment:
Not very clear. Can we add more information here?

Reply:
Updated.